src/ds/d400/d400-private.h

```diff
@@ -341,5 +341,159 @@ namespace librealsense
             { isp_boot_data_upload_failed, "ISP boot data upload failure" },
         };
 
+        class d400_hwmon_response_handler : public base_hwmon_response_handler
+        {
+        public:
+            enum d400_hwmon_response_names : int32_t
+            {
```

Comment:
Rename `d400_hwmon_response_names` to `opcodes`. It will be under the class namespace.